### ISSUE TITLE
impossible to disable user

### ISSUE THREAD

**Issue by paurakhsharma** [owncloud/user_management#24]
## How to reproduce 

1. Install ownCloud from git master branch
2. Install user_management app
3. Create user through user_management app
4. Disable the new user
5. Reload the page and see the user is still enabled.

Browser sends the request to : http://localhost/core/index.php/apps/user_management/user1/enabled
and status code is: 302 Found
![image](https://user-images.githubusercontent.com/20378877/39042816-8d190494-44ab-11e8-87c0-e0f3bff676a0.png)

**Comment by phil-davis** [comment]
Disabling a user works on the webUI in ``stable10`` (which is a good thing)

**Comment by paurakhsharma** [comment]
UI test to disable user #25 should pass after fixing this problem.

**Comment by paurakhsharma** [comment]
@phil-davis Yes it does work on the webUI in `stable10` but it is the case in `master`
